salt/modules/shadow.py

```diff
@@ -202,6 +202,54 @@ def del_password(name):
     return not uinfo['passwd']
 
 
+def lock_password(name):
+    '''
+    .. versionadded:: 2016.3.2
```

Comment:
Can you change the `2016.3.2` value to `Carbon`? That is the code name for the next feature release, which will be based off of the current develop branch.